examples/tmexample.cpp

```diff
@@ -48,7 +47,26 @@ int main()
 		std::cout << "input, prediction of next = " << categoery
 			<< ", " << to_string(pred_category);
 		std::cout << '\n';
+	/* 
```

Comment:
Please revert tmexample.cpp. It is a example program for others to learn from

Reply:
NP